docs/en/sql-reference/functions/ip-address-functions.md

```diff
@@ -316,6 +316,38 @@ Result:
 
 Same as `toIPv4`, but if the IPv4 address has an invalid format, it returns null.
 
+**Syntax**
```

Comment:
I am not sure that is is really needed, as written above - this function is absolutely the same as the example above, nothing changes in arguments as well. Below there is also example with ipv6 - but there it is short without example as it used to be here, so it becomes inconsistent.

Reply:
I think it's needed because these sections can let users know the function behaviors.

To be consistency, related `toIPv6*` functions can do same things as these `toIPv4*` functions.

It's part of the https://github.com/ClickHouse/clickhouse-docs/issues/2022 issue.

Reply:
Ok